recipe/meta.yaml

```diff
@@ -28,22 +26,23 @@ requirements:
     - setuptools
   run:
     - python
-    - click >=6.6
+    - click >=8.0
     - cloudpickle >=1.5.0
     - dask-core {{ version }}
-    - jinja2
+    - jinja2 >=2.10.3
     - locket >=1.0.0
-    - msgpack-python >=0.6.0
+    - msgpack-python >=1.0.0
     - packaging >=20.0
-    - psutil >=5.0
-    - pyyaml
+    - psutil >=5.7.0
+    - pyyaml >=5.3.1
     - setuptools
-    - sortedcontainers !=2.0.0,!=2.0.1
+    - sortedcontainers >=2.0.5
     - tblib >=1.6.0
-    - toolz >=0.8.2
-    - tornado >=6.0.3,<6.2
-    - urllib3
-    - zict >=0.1.3
+    - toolz >=0.10.0
+    - tornado >=6.0.3
+    - urllib3 >=1.24.3
+    - zict >=2.1.0
+    - importlib-metadata >=4.13.0
```

Comment:
importlib-metadata >=4.13.0 is not required https://github.com/dask/distributed/blob/2023.3.2/requirements.txt

Reply:
it is in the setup.py it fails when I remove it

Reply:
done